index.html

```diff
@@ -267,7 +267,7 @@ <h3>Outreachy</h3>
 				<a class="col-md-4" href="https://gssoc.tech" target="_blank" rel="noopener">
 					<img alt="" src="img/projects/girlscriptsummerofcode.jpg" />
 					<h3>GirlScript Summer of Code</h3>
-					<p>3 month long Open Source program during summers, with an aim to help beginners get started with Open Source Development while encouraging diversity.</p>
+					<p>3 month long Open Source program during summers, to help beginners get started with Open Source Development while encouraging diversity.</p>
```

Comment:
adding 'with an aim' really emphasizes the idea to encourage beginners. May be adding back to the sentence makes it more specific of the goal.